msu/systems/registry/load.nut

```diff
@@ -2,12 +2,18 @@ local function includeFile( _file )
 {
 	::includeFile("msu/systems/registry/", _file);
 }
-includeFile("registry_system.nut");
+includeFile("registry_system");
+includeFile("registry_mod_addon");
 
 local system = ::MSU.Class.RegistrySystem();
-
 ::MSU.System.Registry <- system;
 
+includeFile("mod_source");
+includeFile("mod_source_github");
```

Comment:
Not a big deal but probably better to have a `mod_sources` subfolder. But I don't mind if we leave it like this for now.